seed/serializers/derived_columns.py

```diff
@@ -0,0 +1,27 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2021, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+from rest_framework import serializers
+
+from seed.models.derived_columns import DerivedColumn, DerivedColumnParameter
+
+
+class DerivedColumnParameterSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = DerivedColumnParameter
+        fields = '__all__'
+
+
+class DerivedColumnSerializer(serializers.ModelSerializer):
+    parameters = serializers.SerializerMethodField()
+
+    class Meta:
+        model = DerivedColumn
+        exclude = ['source_columns']
+
+    def get_parameters(self, obj):
```

Comment:
Mostly just curious... Where is this used, or is this for something DRF specific?

Reply:
This is DRF specific, `get_<field name>` allows you to provide a function for getting the serialized value for a field. I ended up refactoring this bit in the most recent derived columns PR #2698